feeder/database/models.py

```diff
@@ -125,7 +145,7 @@ async def get(cls, device_hid):
         return results
 
     @classmethod
-    async def report(cls, device_hid: str, voltage: float, usb_power: bool, charging: bool, ir: bool, rssi: int):
+    async def report(cls, *, gateway_hid: str, device_hid: str, voltage: float, usb_power: bool, charging: bool, ir: bool, rssi: int):
```

Comment:
What is the point of adding the args wildcard? It isn't used in the function and we are never passing in any more arguments than specified.

Reply:
Oh, I kind of wanted to make sure named args are used all the time since the number of arguments is so large it would be easy to mix a positional argument. Isn't this all the rage?!

Reply:
This is honestly the first time I have seen it. Which is weird given the amount of Python I write... 🙃

Having now read PEP 3102, ignore me.